sklearn/tests/test_multioutput.py

```diff
@@ -588,3 +588,25 @@ def test_multioutput_estimator_with_fit_params(estimator, dataset):
     estimator.fit(X, y, some_param=some_param)
     for dummy_estimator in estimator.estimators_:
         assert 'some_param' in dummy_estimator._fit_params
+
+
+def test_regressor_chain_w_fit_params():
```

Comment:
Add a small comment e.g.

`# Make sure fit_params are properly propagated to the sub-estimators`